presto-docs/src/main/sphinx/release/release-0.279.2.rst

```diff
@@ -0,0 +1,7 @@
+===============
+Release 0.279.2
+===============
+
+General Changes
```

Comment:
Do we need to add `Upgrade Airlift to 0.207` as it's what https://github.com/prestodb/presto/commit/0325d3d4dffd4a251603544c6eed57e8453c7519 is doing.